BasicTokenList.json

```diff
@@ -4,7 +4,7 @@
     "version": {
         "major": 1,
         "minor": 2,
-        "patch": 4
+        "patch": 5
```

Comment:
When adding new tokens you need to increase minor version. Check the project readme file

Reply:
Thanks, will change to 1.3.0

Reply:
Not sure about lower level version resets when upper level version increases - keeping 1.3.4